lib/tasks/remove_permission_from_role.rake

```diff
@@ -0,0 +1,10 @@
+# to run this rake task run the following command.
+# rake goodcity:remove_permission_from_role permission=permission_name role=role_name
+
+namespace :goodcity do
+  task remove_permission_from_role: :environment do
+    Permission.find_by(name: ENV["permission"])
```

Comment:
I think rake supports command line arguments, maybe more appropriate than env variables. Not a big deal though

Reply:
>command line arguments 

@patrixr , If i run `rake goodcity:remove_permission_from_role permission=abc role=xyz`
These are available as `ENV` variables 
i.e.  
`ENV['permission']="abc"`
`ENV['role']="xyz"`

Reply:
I know, it's fine. It just that ENV variables generally tend to serve a different purpose than args. So it's more "proper" to pass in arguments. But doesn't really make a practical difference

Reply:
no need to change anything, just thought I'd point it out : )